app/views/users/invitation_resendings/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<% content_for :title_caption, "Manage other users" %>
+<% content_for :title, "Resend signup email for #{@user.name}" %>
+
+<% content_for :breadcrumbs,
+   render("govuk_publishing_components/components/breadcrumbs", {
+     collapse_on_mobile: true,
+     breadcrumbs: [
+       {
+         title: "Dashboard",
+         url: root_path,
+       },
+       {
+         title: "Users",
+         url: users_path,
+       },
+       {
+         title: @user.name,
+         url: edit_user_path(@user),
+       },
+       {
+         title: "Resend signup email",
+       }
+     ]
+   })
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for @user, url: user_invitation_resending_path(@user) do %>
+      <%= render "govuk_publishing_components/components/hint", {
+        text: "Invitation not accepted yet. This user hasn't clicked on the link in their signup mail yet."
```

Comment:
Minor: I appreciate this text is copied from the previous flash/info message, but now it's a single paragraph the repeated "yet" feels a bit weird. I'd be tempted to drop the first "yet", i.e. : "Invitation not accepted. This user hasn't clicked on the link in their signup mail yet.". But you could also run this past Etain if you wanted, even after this PR is merged.

Reply:
Addressed in https://github.com/alphagov/signon/pull/2559/commits/a6e562ce5dc98751d454987a2008e7f2227fd893.